utilities.py

```diff
@@ -0,0 +1,35 @@
+def flat(data, parent_dir):
+    '''
+    Convert JSON data to a dictionary.

```

Comment:
This comment needs to be more descriptive. The JSON data is a dictionary. What does this function do to the dictionary and what is it flattening?